src/coreclr/jit/compiler.cpp

```diff
@@ -6108,21 +6092,25 @@ int Compiler::compCompileHelper(CORINFO_MODULE_HANDLE classPtr,
 
     info.compHasNextCallRetAddr = false;
 
+    if (opts.IsReversePInvoke())
+    {
+        info.compCallConv = info.compCompHnd->getUnmanagedCallConv(methodInfo->ftn, nullptr);
```

Comment:
So for reverse P/Invokes the suppression concept isn't needed I guess, but I don't know if that is worth the non-explicit setting of an out.